lib/fxpay/pay.js

```diff
@@ -74,8 +88,8 @@
       paymentWindow.location = payUrl;
 
       function receivePaymentMessage(event) {
-        exports.acceptPayMessage(event, utils.getUrlOrigin(payUrl),
-                                 function(err) {
+
+        var messageCallback = function(err) {
```

Comment:
you could use a named function here for the same effect: ``function messageCallback() {...}``